docs/live-sim.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Live Simulation Mode
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Live Simulation Mode
+
+Forget screen sharing, there’s live simulation mode! Let’s say you’re working on the application logic and want a second opinion or to share your machine with your team mates, live simulation has made this process as easy as sharing a link. It is not necessary to take screenshots anymore because this feature will let all users with the specially generated link to access and discuss in real time the process of your machine.
```

Comment:
This is a really fun opening that I think might work perfectly for a blog post. For docs, we want to imagine people are looking for information quickly; they want to get in, find out, get out. So I’d maybe try to make this as concise as possible and focus on what live simulation mode does. Something like (feel free to do differently!):

```suggestion
Live simulation mode helps you share your simulated machine with your team without screen sharing or screenshots. Share the generated live simulation link, and your team can view your simulation in real-time and even trigger events themselves.
```